<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tunisian Dry Cleaner</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Gestion des employés</string>
     </attribute>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>381</width>
        <height>501</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_ajouter">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>171</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Ajouter un employé :</string>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_nom">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>65</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Nom :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_adresse">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>135</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Adresse :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_prenom">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Prénom :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_tel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>170</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Téléphone :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_fonc">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>240</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Fonction :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_salaire">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>205</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Salaire :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_ev">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>275</y>
          <width>201</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Évaluation de rendement:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nom">
        <property name="geometry">
         <rect>
          <x>115</x>
          <y>60</y>
          <width>230</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_prenom">
        <property name="geometry">
         <rect>
          <x>115</x>
          <y>95</y>
          <width>230</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_adresse">
        <property name="geometry">
         <rect>
          <x>115</x>
          <y>130</y>
          <width>230</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_tel">
        <property name="geometry">
         <rect>
          <x>115</x>
          <y>165</y>
          <width>230</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_salaire">
        <property name="geometry">
         <rect>
          <x>115</x>
          <y>200</y>
          <width>230</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QRadioButton" name="radio1_ev">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>300</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Satisfaisante</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radio2_ev">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>330</y>
          <width>97</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Bonne</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radio3_ev">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>360</y>
          <width>97</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Exellente</string>
        </property>
       </widget>
       <widget class="QPushButton" name="bouton_valider">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>400</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Valider</string>
        </property>
       </widget>
       <widget class="QPushButton" name="bouton_annuler">
        <property name="geometry">
         <rect>
          <x>145</x>
          <y>400</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Annuler</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_ajouter">
        <property name="geometry">
         <rect>
          <x>115</x>
          <y>235</y>
          <width>230</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="bouton_page2">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>0</y>
          <width>75</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Suivant</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_modifier">
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>-10</y>
          <width>341</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Modifier un employé :</string>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_nom_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>65</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Nom :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_prenom_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Prénom :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_tel_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>170</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Téléphone :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_adresse_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>135</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Adresse :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_salaire_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>205</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Salaire :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_fonc_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>240</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Fonction :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_ev_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>275</y>
          <width>201</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Évaluation de rendement:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nom_2">
        <property name="geometry">
         <rect>
          <x>115</x>
          <y>60</y>
          <width>230</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_prenom_2">
        <property name="geometry">
         <rect>
          <x>115</x>
          <y>95</y>
          <width>230</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_adresse_2">
        <property name="geometry">
         <rect>
          <x>115</x>
          <y>130</y>
          <width>230</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_tel_2">
        <property name="geometry">
         <rect>
          <x>115</x>
          <y>165</y>
          <width>230</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_salaire_2">
        <property name="geometry">
         <rect>
          <x>115</x>
          <y>200</y>
          <width>230</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QRadioButton" name="radio1_ev_2">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>275</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Satisfaisante</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radio2_ev_2">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>305</y>
          <width>97</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Bonne</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radio3_ev_2">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>340</y>
          <width>97</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Exellente</string>
        </property>
       </widget>
       <widget class="QPushButton" name="bouton_modifier">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>370</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Ok</string>
        </property>
       </widget>
       <widget class="QPushButton" name="bouton_annuler_mdf">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>370</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Annuler</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_modifier">
        <property name="geometry">
         <rect>
          <x>115</x>
          <y>235</y>
          <width>200</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>395</y>
          <width>191</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Supprimer un employé :</string>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_nom_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>435</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Matricule :</string>
        </property>
       </widget>
       <widget class="QPushButton" name="bouton_supprimer">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>435</y>
          <width>93</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="bouton_page1">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>0</y>
          <width>75</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Précédent</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_nom_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Matricule :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_mat_modif">
        <property name="geometry">
         <rect>
          <x>115</x>
          <y>30</y>
          <width>100</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="bouton_mat_modifier">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>30</y>
          <width>93</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_supression">
        <property name="geometry">
         <rect>
          <x>115</x>
          <y>435</y>
          <width>100</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QTableView" name="tab_employes">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>20</y>
        <width>361</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Gestion des clients</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Gestion des services</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Gestion des fournisseurs</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Gestion des produits</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
